compiler/include/optimizations.h

```diff
@@ -88,4 +89,5 @@ void setDefinedConstForDefExprIfApplicable(DefExpr* defExpr,
 void setDefinedConstForPrimSetMemberIfApplicable(CallExpr *call);
 void setDefinedConstForFieldsInInitializer(FnSymbol *fn);
 
+void gpuTransforms();
```

Comment:
We could create a gpuTransforms.h and put this there. For now, I think it's OK, but `optimizations.h` is not the best place for this, arguably.

Reply:
My thinking was eventually this'll end up in `passes.h`, but I don't want to put it in there until it's a bona-fide pass.

Reply:
Yeah, that's a good point. Leaving this here makes sense.